<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="apples-MacBook-Pro-2.local" name="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings" tests="31" failures="4" timestamp="29 May 2019 11:32:11 GMT" time="308.043" errors="6">
  <testcase name="gettingDashboard" time="0.855" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="4.102" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="2.345" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.719" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="14.223" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:15)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:36)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="2.123" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="2.231" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="20.742" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 20 seconds waiting for presence of element located by: By.id: •••• 8292 
Build info: version: &#039;2.45.0&#039;, revision: &#039;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&#039;, time: &#039;2015-02-27 09:10:26&#039;
System info: host: &#039;apples-MacBook-Pro-2.local&#039;, ip: &#039;192.168.1.138&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;1.8.0_201&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 20 seconds waiting for presence of element located by: By.id: •••• 8292 
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.citi.mobileAutomation.utils.ResuableFunction.getTextFromElementByWait(ResuableFunction.java:178)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:35)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 942 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
*** Element info: {Using=id, value=•••• 8292 }
at sun.reflect.GeneratedConstructorAccessor25.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:67)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:393)
at io.appium.java_client.DefaultGenericMobileDriver.findElementById(DefaultGenericMobileDriver.java:75)
at io.appium.java_client.AppiumDriver.findElementById(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElementById(IOSDriver.java:1)
at org.openqa.selenium.By$ById.findElement(By.java:214)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:63)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:106)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:103)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
... 29 more
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="2.076" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:36)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="2.075" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.637" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="3.042" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="21.020" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 20 seconds waiting for presence of element located by: By.id: Savings Account
Build info: version: &#039;2.45.0&#039;, revision: &#039;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&#039;, time: &#039;2015-02-27 09:10:26&#039;
System info: host: &#039;apples-MacBook-Pro-2.local&#039;, ip: &#039;192.168.1.138&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;1.8.0_201&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 20 seconds waiting for presence of element located by: By.id: Savings Account
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.citi.mobileAutomation.utils.ResuableFunction.getTextFromElementByWait(ResuableFunction.java:178)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:35)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 339 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
*** Element info: {Using=id, value=Savings Account}
at sun.reflect.GeneratedConstructorAccessor25.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:67)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:393)
at io.appium.java_client.DefaultGenericMobileDriver.findElementById(DefaultGenericMobileDriver.java:75)
at io.appium.java_client.AppiumDriver.findElementById(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElementById(IOSDriver.java:1)
at org.openqa.selenium.By$ById.findElement(By.java:214)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:63)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:106)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:103)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
... 29 more
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="0.272" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:18)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="0.785" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.372" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="2.838" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="21.085" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 20 seconds waiting for presence of element located by: By.id: 27 Feb
Build info: version: &#039;2.45.0&#039;, revision: &#039;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&#039;, time: &#039;2015-02-27 09:10:26&#039;
System info: host: &#039;apples-MacBook-Pro-2.local&#039;, ip: &#039;192.168.1.138&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;1.8.0_201&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 20 seconds waiting for presence of element located by: By.id: 27 Feb
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.citi.mobileAutomation.utils.ResuableFunction.getTextFromElementByWait(ResuableFunction.java:178)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:35)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 135 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
*** Element info: {Using=id, value=27 Feb}
at sun.reflect.GeneratedConstructorAccessor25.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:67)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:393)
at io.appium.java_client.DefaultGenericMobileDriver.findElementById(DefaultGenericMobileDriver.java:75)
at io.appium.java_client.AppiumDriver.findElementById(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElementById(IOSDriver.java:1)
at org.openqa.selenium.By$ById.findElement(By.java:214)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:63)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:106)
at org.openqa.selenium.support.ui.ExpectedConditions$3.apply(ExpectedConditions.java:103)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
... 28 more
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="73.298" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="org.openqa.selenium.WebDriverException" message="An unknown server-side error occurred while processing the command. Original error: Unable to parse the format string &quot;value == •••• 8292 &quot;

(
	0   CoreFoundation                      0x000000010dba51bb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000010d143735 objc_exception_throw + 48
	2   Foundation                          0x000000010cc7e157 _qfqp2_performParsing + 7629
	3   Foundation                          0x000000010cc57b03 +[NSPredicate predicateWithFormat:arguments:] + 46
	4   WebDriverAgentLib                   0x0000000124bf137d +[FBPredicate predicateWithFormat:] + 397
	5   WebDriverAgentLib                   0x0000000124c4379e +[FBElementCommands handleScroll:] + 1582
	6   WebDriverAgentLib                   0x0000000124c0f196 -[FBRoute_TargetAction mountRequest:intoResponse:] + 182
	7   WebDriverAgentLib                   0x0000000124bfc029 __37-[FBWebServer registerRouteHandlers:]_block_invoke + 553
	8   RoutingHTTPServer                   0x0000000124cf6c18 -[RoutingHTTPServer handleRoute:withRequest:response:] + 136
	9   RoutingHTTPServer                   0x0000000124cf74fa __72-[RoutingHTTPServer routeMethod:withPath:parameters:request:connection:]_block_invoke + 47
	10  libdispatch.dylib                   0x000000011262f54b _dispatch_client_callout + 8
	11  libdispatch.dylib                   0x000000011263c036 _dispatch_async_and_wait_invoke + 109
	12  libdispatch.dylib                   0x000000011262f54b _dispatch_client_callout + 8
	13  libdispatch.dylib                   0x000000011263b380 _dispatch_main_queue_callback_4CF + 1290
	14  CoreFoundation                      0x000000010db0a3e9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	15  CoreFoundation                      0x000000010db04a76 __CFRunLoopRun + 2342
	16  CoreFoundation                      0x000000010db03e11 CFRunLoopRunSpecific + 625
	17  Foundation                          0x000000010cbc7322 -[NSRunLoop(NSRunLoop) runMode:beforeDate:] + 277
	18  WebDriverAgentLib                   0x0000000124bfa920 -[FBWebServer startServing] + 368
	19  WebDriverAgentRunner                0x0000000124be703a -[UITestingUITests testRunner] + 122
	20  CoreFoundation                      0x000000010dbac03c __invoking___ + 140
	21  CoreFoundation                      0x000000010dba94d5 -[NSInvocation invoke] + 325
	22  XCTest                              0x000000010c9a9018 __24-[XCTestCase invokeTest]_block_invoke.196 + 78
	23  XCTest                              0x000000010ca00c74 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
	24  XCTest                              0x000000010ca00b91 -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
	25  XCTest                              0x000000010c9a8cd9 __24-[XCTestCase invokeTest]_block_invoke + 848
	26  XCTest                              0x000000010ca06b7e -[XCUITestContext performInScope:] + 248
	27  XCTest                              0x000000010c9a88ce -[XCTestCase testContextPerformInScope:] + 98
	28  XCTest                              0x000000010c9a897c -[XCTestCase invokeTest] + 137
	29  XCTest                              0x000000010c9aa4b7 __26-[XCTestCase performTest:]_block_invoke_2 + 43
	30  XCTest                              0x000000010ca00c74 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
	31  XCTest                              0x000000010ca00b91 -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
	32  XCTest                              0x000000010c9aa3ce __26-[XCTestCase performTest:]_block_invoke.330 + 88
	33  XCTest                              0x000000010ca1144b +[XCTContext runInContextForTestCase:block:] + 225
	34  XCTest                              0x000000010c9a9afd -[XCTestCase performTest:] + 675
	35  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
	36  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
	37  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
	38  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
	39  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
	40  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
	41  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
	42  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
	43  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
	44  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
	45  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
	46  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
	47  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
	48  XCTest                              0x000000010ca1ce86 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke + 171
	49  XCTest                              0x000000010ca1cfa7 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke.80 + 68
	50  XCTest                              0x000000010c9bcbc1 -[XCTestObservationCenter _observeTestExecutionForBlock:] + 585
	51  XCTest                              0x000000010ca1cbfa -[XCTTestRunSession runTestsAndReturnError:] + 623
	52  XCTest                              0x000000010c9896b6 -[XCTestDriver runTestsAndReturnError:] + 422
	53  XCTest                              0x000000010ca0d9cd _XCTestMain + 1478
	54  WebDriverAgentRunner-Runner         0x000000010c6a802d -[_XCTRunnerAppDelegate applicationWillResignActive:] + 0
	55  WebDriverAgentRunner-Runner         0x000000010c6a7f2b _XCTRunnerRunTests + 0
	56  CoreFoundation                      0x000000010db0a62c __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
	57  CoreFoundation                      0x000000010db09de0 __CFRunLoopDoBlocks + 336
	58  CoreFoundation                      0x000000010db04654 __CFRunLoopRun + 1284
	59  CoreFoundation                      0x000000010db03e11 CFRunLoopRunSpecific + 625
	60  GraphicsServices                    0x00000001161911dd GSEventRunModal + 62
	61  UIKitCore                           0x000000010eb3881d UIApplicationMain + 140
	62  WebDriverAgentRunner-Runner         0x000000010c6a81f5 main + 183
	63  libdyld.dylib                       0x000000011269f575 start + 1
	64  ???                                 0x0000000000000005 0x0 + 5
) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 432 milliseconds
Build info: version: &#039;2.45.0&#039;, revision: &#039;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&#039;, time: &#039;2015-02-27 09:10:26&#039;
System info: host: &#039;apples-MacBook-Pro-2.local&#039;, ip: &#039;192.168.1.138&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;1.8.0_201&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be">
      <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Unable to parse the format string "value == •••• 8292 "

(
0   CoreFoundation                      0x000000010dba51bb __exceptionPreprocess + 331
1   libobjc.A.dylib                     0x000000010d143735 objc_exception_throw + 48
2   Foundation                          0x000000010cc7e157 _qfqp2_performParsing + 7629
3   Foundation                          0x000000010cc57b03 +[NSPredicate predicateWithFormat:arguments:] + 46
4   WebDriverAgentLib                   0x0000000124bf137d +[FBPredicate predicateWithFormat:] + 397
5   WebDriverAgentLib                   0x0000000124c4379e +[FBElementCommands handleScroll:] + 1582
6   WebDriverAgentLib                   0x0000000124c0f196 -[FBRoute_TargetAction mountRequest:intoResponse:] + 182
7   WebDriverAgentLib                   0x0000000124bfc029 __37-[FBWebServer registerRouteHandlers:]_block_invoke + 553
8   RoutingHTTPServer                   0x0000000124cf6c18 -[RoutingHTTPServer handleRoute:withRequest:response:] + 136
9   RoutingHTTPServer                   0x0000000124cf74fa __72-[RoutingHTTPServer routeMethod:withPath:parameters:request:connection:]_block_invoke + 47
10  libdispatch.dylib                   0x000000011262f54b _dispatch_client_callout + 8
11  libdispatch.dylib                   0x000000011263c036 _dispatch_async_and_wait_invoke + 109
12  libdispatch.dylib                   0x000000011262f54b _dispatch_client_callout + 8
13  libdispatch.dylib                   0x000000011263b380 _dispatch_main_queue_callback_4CF + 1290
14  CoreFoundation                      0x000000010db0a3e9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
15  CoreFoundation                      0x000000010db04a76 __CFRunLoopRun + 2342
16  CoreFoundation                      0x000000010db03e11 CFRunLoopRunSpecific + 625
17  Foundation                          0x000000010cbc7322 -[NSRunLoop(NSRunLoop) runMode:beforeDate:] + 277
18  WebDriverAgentLib                   0x0000000124bfa920 -[FBWebServer startServing] + 368
19  WebDriverAgentRunner                0x0000000124be703a -[UITestingUITests testRunner] + 122
20  CoreFoundation                      0x000000010dbac03c __invoking___ + 140
21  CoreFoundation                      0x000000010dba94d5 -[NSInvocation invoke] + 325
22  XCTest                              0x000000010c9a9018 __24-[XCTestCase invokeTest]_block_invoke.196 + 78
23  XCTest                              0x000000010ca00c74 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
24  XCTest                              0x000000010ca00b91 -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
25  XCTest                              0x000000010c9a8cd9 __24-[XCTestCase invokeTest]_block_invoke + 848
26  XCTest                              0x000000010ca06b7e -[XCUITestContext performInScope:] + 248
27  XCTest                              0x000000010c9a88ce -[XCTestCase testContextPerformInScope:] + 98
28  XCTest                              0x000000010c9a897c -[XCTestCase invokeTest] + 137
29  XCTest                              0x000000010c9aa4b7 __26-[XCTestCase performTest:]_block_invoke_2 + 43
30  XCTest                              0x000000010ca00c74 -[XCTestCase(Failures) performFailableBlock:testCaseRun:shouldInterruptTest:] + 57
31  XCTest                              0x000000010ca00b91 -[XCTestCase(Failures) _performTurningExceptionsIntoFailuresInterruptAfterHandling:block:] + 96
32  XCTest                              0x000000010c9aa3ce __26-[XCTestCase performTest:]_block_invoke.330 + 88
33  XCTest                              0x000000010ca1144b +[XCTContext runInContextForTestCase:block:] + 225
34  XCTest                              0x000000010c9a9afd -[XCTestCase performTest:] + 675
35  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
36  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
37  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
38  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
39  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
40  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
41  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
42  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
43  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
44  XCTest                              0x000000010c9a4ccb __27-[XCTestSuite performTest:]_block_invoke + 365
45  XCTest                              0x000000010c9a44a3 -[XCTestSuite _performProtectedSectionForTest:testSection:] + 55
46  XCTest                              0x000000010c9a4766 -[XCTestSuite performTest:] + 296
47  XCTest                              0x000000010c9ed1a2 -[XCTest runTest] + 57
48  XCTest                              0x000000010ca1ce86 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke + 171
49  XCTest                              0x000000010ca1cfa7 __44-[XCTTestRunSession runTestsAndReturnError:]_block_invoke.80 + 68
50  XCTest                              0x000000010c9bcbc1 -[XCTestObservationCenter _observeTestExecutionForBlock:] + 585
51  XCTest                              0x000000010ca1cbfa -[XCTTestRunSession runTestsAndReturnError:] + 623
52  XCTest                              0x000000010c9896b6 -[XCTestDriver runTestsAndReturnError:] + 422
53  XCTest                              0x000000010ca0d9cd _XCTestMain + 1478
54  WebDriverAgentRunner-Runner         0x000000010c6a802d -[_XCTRunnerAppDelegate applicationWillResignActive:] + 0
55  WebDriverAgentRunner-Runner         0x000000010c6a7f2b _XCTRunnerRunTests + 0
56  CoreFoundation                      0x000000010db0a62c __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
57  CoreFoundation                      0x000000010db09de0 __CFRunLoopDoBlocks + 336
58  CoreFoundation                      0x000000010db04654 __CFRunLoopRun + 1284
59  CoreFoundation                      0x000000010db03e11 CFRunLoopRunSpecific + 625
60  GraphicsServices                    0x00000001161911dd GSEventRunModal + 62
61  UIKitCore                           0x000000010eb3881d UIApplicationMain + 140
62  WebDriverAgentRunner-Runner         0x000000010c6a81f5 main + 183
63  libdyld.dylib                       0x000000011269f575 start + 1
64  ???                                 0x0000000000000005 0x0 + 5
) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 432 milliseconds
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'apples-MacBook-Pro-2.local', ip: '192.168.1.138', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_201'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities [{app=/Users/apple/Desktop/UK App/CN_UAT2_27FEB_DEBUG.app, networkConnectionEnabled=false, bundleId=com.citi.development, databaseEnabled=false, deviceName=iPhone 8 Plus, platform=MAC, newCommandTimeout=3000, platformVersion=12.1, webStorageEnabled=false, locationContextEnabled=false, browserName=, takesScreenshot=true, javascriptEnabled=true, platformName=iOS, udid=7E03F64A-441C-44FE-BCB5-E89550252330, autoAcceptAlerts=true}]
Session ID: e254d904-085d-4b0d-9abe-69aaf69f95be
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:508)
at com.citi.mobileAutomation.utils.ResuableFunction.scrolliOS(ResuableFunction.java:593)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:54)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="0.670" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.636" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.394" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:18)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="0.865" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.712" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="4.932" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.273" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="2.588" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:36)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="1.909" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="19.005" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:36)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:75)
at com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings.gettingDashboard(Test_EMEA_UK_AD_100_Savings.java:23)
at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="100.996" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
  <testcase name="gettingDashboard" time="0.223" classname="com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings"/>
</testsuite> <!-- com.citi.mobileAutomation.test_UK_iOS.Test_EMEA_UK_AD_100_Savings -->
